label_semantic_roles/README_en.md

```diff
@@ -1,6 +1,8 @@
 # Semantic Role Labeling
 
-本教程源代码目录在[book/label_semantic_roles](https://github.com/PaddlePaddle/book/tree/develop/label_semantic_roles)， 初次使用请参考PaddlePaddle[安装教程](http://www.paddlepaddle.org/doc_cn/build_and_install/index.html)。
+The source code of this chapter lives on [book/label_semantic_roles](https://github.com/PaddlePaddle/book/tree/develop/label_semantic_roles).
```

Comment:
on a server, in a box. Not sure what URLs should be/ what reads better. Let's say 'accessible at' instead?